pyperipheral3 breaking libpanda-mipsel build

```
  LINK    mipsel-softmmu/libpanda-mipsel.so
/usr/bin/ld: /home/luke/.pyenv/versions/3.9.0/lib/python3.9/config-3.9-x86_64-linux-gnu/libpython3.9.a(_warnings.o): relocation R_X86_64_PC32 against symbol `_PyRuntime' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: final link failed: Bad value
collect2: error: ld returned 1 exit status
Makefile:235: recipe for target 'libpanda-mipsel.so' failed
make[1]: *** [libpanda-mipsel.so] Error 1
Makefile:357: recipe for target 'subdir-mipsel-softmmu' failed
make: *** [subdir-mipsel-softmmu] Error 2
```

This happens when pyperipheral3 is enabled. Not sure why it's just MIPSEL.

--- a/configure
+++ b/configure
@@ -415,7 +415,7 @@ jemalloc="no"
 replication="yes"
 #AVATAR
 pyperipheral2="no"
-pyperipheral3="yes"
+pyperipheral3="no"
 
 
 supported_cpu="no"

